docs/api.md

```diff
@@ -1246,6 +1249,35 @@ puppeteer.launch().then(async browser => {
 ```
 Shortcut for [page.mainFrame().waitForSelector(selector[, options])](#framewaitforselectorselector-options).
 
+#### page.waitForXpath(xpath[, options])
+- `xpath` <[string]> A [xpath] of an element to wait for,
```

Comment:
The bottom reference for `[xpath]` link is missing.

Reply:
ah, missed that. added the link.